build.gradle

```diff
@@ -0,0 +1,30 @@
+buildscript {
+    repositories {
+        mavenLocal()
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.mparticle:android-kit-plugin:+'
+    }
+}
+
+apply plugin: 'com.mparticle.kit'
+
+repositories {
+    maven {
+        url 'https://artifactory.roko.mobi/artifactory/libs-release'
+    }
+}
+
+android {
+    defaultConfig {
+        minSdkVersion 16
+    }
+}
+
+dependencies {
+    compile 'com.android.support:appcompat-v7:24.2.1'
+    compile 'com.android.support:recyclerview-v7:24.2.1'
+    compile 'com.rokolabs:rokomobi-android-sdk:1.12.+@aar'
```

Comment:
For Android we unfortunately need to lock versions, cannot use wildcards for transitive dependencies. It will yield unpredictable builds for customers. iOS is a different story due to podfile.lock

Reply:
makes sense. we'll remove wildcard and update to include full version number. 

additionally, we'll submit future PRs with updated version string whenever we release a new SDK on our side.